CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - [#1197](https://github.com/crypto-org-chain/cronos/pull/1197) tune rocksdb options to control memory consumption.
 - [#1207](https://github.com/crypto-org-chain/cronos/pull/1207) Update rocksdb to `v8.6.7`.
+- [#](https://github.com/crypto-org-chain/cronos/pull/) Fill default value for staking gensis to avoid panic in pruned node.
```

Comment:
```suggestion
- [#1225](https://github.com/crypto-org-chain/cronos/pull/1225) Fill default value for staking gensis to avoid panic in pruned node.
```